clio-client/src/main/scala/org/broadinstitute/clio/client/commands/Executor.scala

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.clio.client.commands
+
+import akka.http.scaladsl.model.HttpResponse
+import org.broadinstitute.clio.client.webclient.ClioWebClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait Executor {
+  def execute(webClient: ClioWebClient, bearerToken: String)(
```

Comment:
Could we have a more restricted type for `bearerToken`? It seems like we don't need to perform string operations on it between its creation and its usage.

Is there a way to use an implicit to avoid having to pass it around? It will be the same for all method calls made during the lifetime of JVM, correct?

Reply:
yup